
package com.laegler.openbanking.soap.service;

import javax.xml.ws.WebFault;


/**
 * Bad Request
 *
 * This class was generated by Apache CXF 3.3.3
 * 2019-10-21T07:01:27.394+13:00
 * Generated source version: 3.3.3
 */

@WebFault(name = "PUT_Updatethebusinessorthetechnicalcontactsforthegivenorganisation_400", targetNamespace = "http://laegler.com/openbanking/soap/model")
public class PUTUpdatethebusinessorthetechnicalcontactsforthegivenorganisation400 extends Exception {

    private java.lang.Object putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation400;

    public PUTUpdatethebusinessorthetechnicalcontactsforthegivenorganisation400() {
        super();
    }

    public PUTUpdatethebusinessorthetechnicalcontactsforthegivenorganisation400(String message) {
        super(message);
    }

    public PUTUpdatethebusinessorthetechnicalcontactsforthegivenorganisation400(String message, java.lang.Throwable cause) {
        super(message, cause);
    }

    public PUTUpdatethebusinessorthetechnicalcontactsforthegivenorganisation400(String message, java.lang.Object putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation400) {
        super(message);
        this.putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation400 = putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation400;
    }

    public PUTUpdatethebusinessorthetechnicalcontactsforthegivenorganisation400(String message, java.lang.Object putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation400, java.lang.Throwable cause) {
        super(message, cause);
        this.putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation400 = putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation400;
    }

    public java.lang.Object getFaultInfo() {
        return this.putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation400;
    }
}
